hack/monitor_prow.py

```diff
@@ -42,28 +48,92 @@ def highlight(log_dir, dc):
     error = '"level":"error"'
     fatal = '"level":"fatal"'
     log = '{}/{}.log'.format(log_dir, dc)
-    header = '{}{}:{}'.format(BOLD, dc, RESET)
     while True:
+        debug("deployment/{}: gathering info".format(dc))
+        header, deployError = renderHeader(dc)
         lines = []
+        log_lines = []
         for pod in exec_cmd(
                 'oc', 'get', 'pods', '--namespace', 'ci',
                 '--selector', 'component={}'.format(dc),
                 '--output', 'jsonpath={.items[*].metadata.name}').split():
+            debug("deployment/{}: pod/{}: gathering info".format(dc, pod))
+            lines.append(*renderFlavor(pod, dc))
```

Comment:
`list.append` doesn't handle the zeroth case, we will get

`TypeError: append() takes exactly one argument (0 given)`

Reply:
```suggestion
            lines.extend(renderFlavor(pod, dc))
```

Reply:
wowza